addons/gitlab/api.py

```diff
@@ -78,9 +84,11 @@ def branches(self, repo_id, branch=None):
         :return: List of branch dicts
         """
         if branch:
-            return gitlab.project.get(self.gitlab, repo_id).branches.get(branch)
+            with catch_token_error():
+                return gitlab.project.get(self.gitlab, repo_id).branches.get(branch)
 
-        return gitlab.project.get(self.gitlab, repo_id).branches.list()
+        with catch_token_error():
+            return gitlab.project.get(self.gitlab, repo_id).branches.list()
```

Comment:
`self.gitlab.projects.get(repo_id).branches.list()`? This is the line raising an `AttributeError`, likely similar above.